### PR TITLE
Update RELEASE.md

### PR DESCRIPTION
**Pull request by nathanhammond**
I need to remember to check the experiments stuff during the next release. This addition is so that future me doesn't screw up a release and have to do it twice.

### CODE REVIEW

File: RELEASE.md [hunk 1]
```diff
@@ -94,6 +94,10 @@ git checkout -B master --track origin/master
 
 # Merge the new beta release into the "master" branch
 git merge vX.Y.0-beta.1
+
+# Confirm that experiments are turned on.
```
Comment (kellyselden) [inline RELEASE.md]: How do you confirm? For future non-nathans

Comment (hjdivad) [inline RELEASE.md]: @kellyselden `Object.keys(require('ember-cli/lib/experiments')).length` or similar

**Comment by kellyselden** [comment]
It reads to me the first warning says in beta experiments are on, then the second warning says in beta experiments are off.

**Comment by hjdivad** [comment]
> It reads to me the first warning says in beta experiments are on, then the second warning says in beta experiments are off.

It reads the same way to me.  "Promoting canary to beta" should ensure experiments are off.  Releasing canary is the only time we should expect experiments to be on.

**Comment by nathanhammond** [comment]
> It reads the same way to me. "Promoting canary to beta" should ensure experiments are off. Releasing canary is the only time we should expect experiments to be on.

This is the portion of directions at the tail end, _after_ the release, syncing the state back up.

**Comment by nathanhammond** [comment]
@homu r+

**Pull request merged**